sympy/control/core.py

```diff
@@ -0,0 +1,269 @@
+"""
+A symbolic control toolbox.
+"""
+
+import sympy
+import mpmath
+
+# pylint: disable=invalid-name
+
+s, t = sympy.symbols('s, t')
+
+def array_to_poly(a, s):
+    """
+    Convert an array into a polynomial in s
+
+    :param a: array
+    :param x: variable
+    """
+    n = len(a)
+    p = 0
+    for i, c in enumerate(a):
+        p += c*s**(n - i - 1)
+    return p
```

Comment:
It might be a good idea to create a TransferFunction object that utlizes the poly module behind the scenes. The current method proposed here seems to be simply a sympy expression.